core/meta/inc/TEnum.h

```diff
@@ -37,6 +37,7 @@ class TEnum : public TDictionary {
    ClassInfo_t *fInfo;          //!interpreter information, owned by TEnum
    TClass      *fClass;         //!owning class
    std::string  fQualName;      // fully qualified type name
+   EDataType    fUnderlyingType;// Type (size) used to store the enum in memory
```

Comment:
```suggestion
   EDataType    fUnderlyingType = kInt;// Type (size) used to store the enum in memory
```

Reply:
Humm .. other members are not going using this syntax yet.

Reply:
I still think it's the more robust syntax, even for gradual changes, and even if introduced gradually. E.g. it doesn't require the reviewer to check whether you have adapted all constructors ;-)